jwql/utils/utils.py

```diff
@@ -387,20 +389,31 @@ def get_config():
     """
     config_file_location = os.path.join(__location__, 'config.json')
 
+    # Make sure the file exists
     if not os.path.isfile(config_file_location):
         raise FileNotFoundError('The JWQL package requires a configuration file (config.json) '
                                 'to be placed within the jwql/utils directory. '
                                 'This file is missing. Please read the relevant wiki page '
                                 '(https://github.com/spacetelescope/jwql/wiki/'
                                 'Config-file) for more information.')
 
-    with open(config_file_location, 'r') as config_file:
-        settings = json.load(config_file)
+    with open(config_file_location, 'r') as config_file_object:
+        try:
+            # Load it with JSON
+            settings = json.load(config_file_object)
+        except json.JSONDecodeError as e:
+            # Raise a more helpful error if there is a formatting problem
+            raise ValueError('Incorrectly formatted config.json file. '
+                             'Please fix JSON formatting: {}'
+                                       .format(e))
```

Comment:
It looks like this line is indented too much?  Perhaps this can just be put back into line 407.